plugins/modules/aci_bd_rogue_exception_mac.py

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Samita Bhattacharjee (@samitab) <samitab@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_bd_rogue_exception_mac
+short_description: Manage Rogue Exception MAC (fv:RogueExceptionMac)
+description:
+- Manage Rogue Exception MACs in BD's on Cisco ACI fabrics.
+- Only available in APIC version 5.2 or later.
+options:
+  bd:
+    description:
+    - The name of the Bridge Domain.
+    type: str
+    aliases: [ bd_name, bridge_domain ]
+  tenant:
+    description:
+    - The name of the Tenant.
+    type: str
+    aliases: [ tenant_name ]
+  mac:
+    description:
+    - MAC address to except from Rogue processing.
+    type: str
+  description:
+    description:
+    - The description for the Rogue Exception MAC.
+    type: str
+    aliases: [ descr ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+
+notes:
+- The C(tenant) and C(bd) used must exist before using this module in your playbook.
+  The M(cisco.aci.aci_tenant) module and M(cisco.aci.aci_bd) can be used for these.
+seealso:
+- module: cisco.aci.aci_bd
+- module: cisco.aci.aci_tenant
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC class B(fv:RogueExceptionMac).
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Samita Bhattacharjee (@samitab)
+"""
+
+EXAMPLES = r"""
+- name: Create a Tenant
+  cisco.aci.aci_tenant:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    tenant: production
+    state: present
+  delegate_to: localhost
+
+- name: Create a Bridge Domain
+  cisco.aci.aci_bd:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    tenant: production
+    bd: database
+    state: present
+  delegate_to: localhost
```

Comment:
I don't think we include parent configuration into the example section? Has this been discussed as new addition to all examples?

Reply:
Removed